scheme/common/cca/realm/cca_realm_attributes.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cca
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/services/log"
+)
+
+type RealmAttributes struct {
+	Rim       *[]byte
```

Comment:
Shouldn't the initialisms be capilatised, i.e. `RIM`, `REM`, `RPV` (to be consustent with things like `UUID`, `HashAlgID` etc..)?

Reply:
Will do it now!